creation/web_base/condor_startup.sh

```diff
@@ -456,11 +463,12 @@ do
     set_var $line
 done < condor_vars.lst.tmp
 
-
+# The exec command GLIDEIN_WRAPPER_EXEC is and must be already quoted (e.g. "\\\"\\\$@\\\"")
```

Comment:
Is a word missing after the words ... GLIDEIN_WRAPPER_EXEC is but before and must be...? Or is that intentional to convey the message that it is already quoted and the expectation is that it must already be quoted?